front/app/containers/Admin/messaging/AutomatedEmails/ExampleModal/EmptyState.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+// Components
+import { Box, Button, Text } from '@citizenlab/cl2-component-library';
+
+// i18n
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from '../../messages';
+
+const EmptyState = () => {
+  const { formatMessage } = useIntl();
+  return (
+    <Box mb="24px">
+      <Text>
+        <FormattedMessage {...messages.noExampleYet} />
+      </Text>
+      <Box display="flex" justifyContent="center" mt="32px">
+        <a
+          href={formatMessage(messages.supportButtonLink)}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <Button>
+            <FormattedMessage {...messages.supportButtonLabel} />
+          </Button>
+        </a>
+      </Box>
+    </Box>
```

Comment:
Our Button component actually has a linkTo prop :-)

Reply:
I've looked for it, but seems like it's not there, or I can't find it. At least not in the component library, it is there in our local components/Button. Should I use that one instead of component library?

Reply:
Yes, the one from the UI folder needs to be used for now.